media/js/firefox/accounts.js

```diff
@@ -20,6 +22,11 @@
         fxaIframeSrc = fxaIframeSrc.replace('context=iframe', 'context=fx_firstrun_v2');
     }
 
+    // if email address provided in querystring, send to FxA frame
+    if (params.get('fxa-email')) {
+        fxaIframeSrc += '&email=' + params.get('fxa-email');
```

Comment:
@jpetto - [email is an officially supported parameter](https://github.com/mozilla/fxa-content-server/blob/master/docs/query-params.md#email? Before I say "go ahead!" what's feature/goal?